cloudbuild/cloudbuild.yaml

```diff
@@ -0,0 +1,5 @@
+steps:
+  - name:
+    args: ["./cloudbuild/install_requirements.sh"]
```

Comment:
I don't see what this is for? This all happens in the docker image so none of it should be necessary (it will also be slow)

Reply:
Fair point, have removed that.

This command seems to work pretty well: `docker run -it -v /usr/local/google/home/eliaskassell/Documents/github/dataform:/dataform -v ~/.config/gcloud:/root/.config/gcloud -w /dataform --entrypoint bash gcr.io/cloud-builders/bazel:5.4.0 ./scripts/run_tests` - have updated the cloudbuild commands to do this instead.